libs/oidc/provider/src/lib/services/common/common.service.ts

```diff
@@ -1,8 +1,9 @@
-import { getConnection, EntitySchema, FindConditions, BaseEntity } from 'typeorm';
+import { getConnection, EntitySchema, FindConditions } from 'typeorm';
 
-export type TypeORMEntities = string | Function | (new () => unknown) | EntitySchema<unknown>;
+export type TypeORMEntities = string | (new () => unknown) | EntitySchema<unknown>;
 
 export abstract class CommonService {
+  // TODO: Is this even used anymore? Everything looks very shitty - Aaron H (4/21/22)
```

Comment:
`Ctrl + Shift + F12` on member name. Doesn't appear to be used anywhere, so probably not used anymore?